src/main/create-operator-json/core/operatorIdEnum.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+export default {
+    alibi: 1,
+    amaru: 2,
+    ash: 3,
+    bandit: 4,
+    blackbeard: 5,
+    blitz: 6,
+    buck: 7,
+    capitao: 8,
+    castle: 9,
+    caviera: 10,
+    clash: 11,
+    doc: 12,
+    dokkaebi: 13,
+    ela: 14,
+    echo: 15,
+    finka: 16,
+    frost: 17,
+    fuze: 18,
+    glaz: 19,
+    goyo: 20,
+    gridlock: 21,
+    hibana: 22,
+    iq: 23,
+    jackal:24,
+    jager: 25,
+    kaid: 26,
+    kapkan: 27,
+    lesion: 28,
```

Comment:
As new operators come out, we should be able to keep this list alphabetical? All we will have to do is update the numbers in this file?

Reply:
I did alphabetically as it was easier for me to debug for names.

Adding new operators in the list alphabetically could lead to mistakes,
I can change the order to be operator release dates rather than by name.